html/pfappserver/root/static.alt/src/views/Configuration/_components/AuthenticationSourcesView.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <pf-config-view
+    :isLoading="isLoading"
+    :form="getForm"
+    :model="source"
+    :validation="$v.source"
+    :isNew="isNew"
+    :isClone="isClone"
+    @validations="sourceValidations = $event"
+    @close="close"
+    @create="create"
+    @save="save"
+    @remove="remove"
+  >
+    <template slot="header" is="b-card-header">
+      <b-button-close @click="close" v-b-tooltip.hover.left.d300 :title="$t('Close [ESC]')"><icon name="times"></icon></b-button-close>
+      <h4 class="mb-0">
+        <span v-if="!isNew && !isClone">{{ $t('Authentication Source {id}', { id: id }) }}</span>
+        <span v-else-if="isClone">{{ $t('Clone Authentication Source {id}', { id: id }) }}</span>
+        <span v-else>{{ $t('New {sourceType} Authentication Source', { sourceType: this.sourceType}) }}</span>
+      </h4>
+    </template>
+    <template slot="footer" is="b-card-footer" @mouseenter="$v.source.$touch()">
+      <pf-button-save v-if="!isNew && !isClone" :disabled="invalidForm" :isLoading="isLoading" :icon="(ctrlKey) ? 'step-backward' : ''">{{ $t('Save') }}</pf-button-save>
+      <pf-button-save v-else-if="isClone" :disabled="invalidForm" :isLoading="isLoading" :icon="(ctrlKey) ? 'step-backward' : ''">{{ $t('Clone') }}</pf-button-save>
+      <pf-button-save v-else :disabled="invalidForm" :isLoading="isLoading" :icon="(ctrlKey) ? 'step-backward' : ''">{{ $t('Create') }}</pf-button-save>
```

Comment:
You could move the condition logic in the button's label only and keep a single component.